src/styled.js

```diff
@@ -27,18 +33,25 @@ const styled = ({tagName, elementStyle, mountSheet}: StyledArgs) => {
   return class StyledElement extends Component {
     static tagName: string = tagName
     static style: ComponentStyleType = elementStyle
+    static contextTypes = {
+      [channel]: PropTypes.object
+    }
```

Comment:
it's an open question for me - do we really need styles creator for styled-jss, because I don't see performance issues for the current solution (we have an 'overhead' with Object.assign in update, but it's fast enough I think)